src/spritesheet/spritesheetAsset.ts

```diff
@@ -148,7 +158,16 @@ export const spritesheetAsset = {
             {
                 const imagePath = copySearchParams(basePath + (imageFilename ?? asset.meta.image), options.src);
 
-                const assets = await loader.load<Texture>([imagePath]);
+                let assets;
+
+                if (textureOptions)
```

Comment:
I don't think we even need the `if (textureOptions)` check here as textureOptions can be null:

replacing the original line
`assets = await loader.load<Texture>([imagePath]);`

with
`assets = await loader.load<Texture>([{ src: imagePath, data: textureOptions}]);`

should work!

Reply:
It looks like that check is still needed, removing it caused the SVG spritesheet test to fail. 
I could modify the SVG script to fix that issue but I don't know if that's in the scope of this PR, what are your thoughts?